webpack.config.js

```diff
@@ -1,80 +1,99 @@
 const
-  path = require('path'),
-  webpack = require('webpack'),
-  ExtractTextPlugin = require('extract-text-webpack-plugin'),
-  HtmlWebpackPlugin = require('html-webpack-plugin'),
-  extractStylus = 'css-loader?sourceMap!stylus-loader',
-  development = !['production', 'ci'].includes(process.env.NODE_ENV)
+	path = require('path'),
+	webpack = require('webpack'),
+	ExtractTextPlugin = require('extract-text-webpack-plugin'),
+	HtmlWebpackPlugin = require('html-webpack-plugin'),
+	extractStylus = 'css-loader?sourceMap!stylus-loader',
+	MiniCssExtractPlugin = require('mini-css-extract-plugin'),
+	development = !['production', 'ci'].includes(process.env.NODE_ENV)
```

Comment:
please revert the indents back to 2 spaces.

Reply:
@nathanboktae Done 👍